GitVersion is a dependency that needs git history to return the current version.
APKBUILD does not use git history.
Disable it and read from package project Version instead.

--- a/src/Recyclarr.Cli/Console/CliSetup.cs
+++ b/src/Recyclarr.Cli/Console/CliSetup.cs
@@ -2,6 +2,7 @@ using Autofac;
 using Recyclarr.Cli.Console.Commands;
 using Spectre.Console;
 using Spectre.Console.Cli;
+using System.Reflection;
 
 namespace Recyclarr.Cli.Console;
 
@@ -72,7 +73,7 @@ public static class CliSetup
 
             config.SetApplicationName("recyclarr");
             config.SetApplicationVersion(
-                $"v{GitVersionInformation.SemVer} ({GitVersionInformation.FullBuildMetaData})"
+                $"v{Assembly.GetEntryAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion}"
             );
 
             AddCommands(config);

--- a/src/Recyclarr.Cli/Console/Setup/ProgramInformationDisplayTask.cs
+++ b/src/Recyclarr.Cli/Console/Setup/ProgramInformationDisplayTask.cs
@@ -1,5 +1,6 @@
 using Recyclarr.Cli.Console.Commands;
 using Recyclarr.Platform;
+using System.Reflection;
 
 namespace Recyclarr.Cli.Console.Setup;
 
@@ -7,7 +8,7 @@ public class ProgramInformationDisplayTask(ILogger log, IAppPaths paths) : IGlob
 {
     public void OnStart(BaseCommandSettings cmd)
     {
-        log.Debug("Recyclarr Version: {Version}", GitVersionInformation.InformationalVersion);
+        log.Debug("Recyclarr Version: {Version}", Assembly.GetEntryAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion);
         log.Debug("App Data Dir: {AppData}", paths.AppDataDirectory);
     }
 
